python/cuml/internals/api_context_managers.py

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2020-2022, NVIDIA CORPORATION.
+# Copyright (c) 2020-2021, NVIDIA CORPORATION.
```

Comment:
Don't forget to fix this up before merge. Pretty sure this is a bug in our copyright update script.

Reply:
Done.